apps/docs/pages/guides/getting-started/tutorials/with-sveltekit.mdx

```diff
@@ -69,249 +69,380 @@ Install the auth helpers for SvelteKit:
+### Set up a Login page
 
-```ts title=src/hooks.client.ts
-import '$lib/supabaseClient'
-```
+#### Supabase Auth UI
 
-```ts title=src/hooks.server.ts
-import '$lib/supabaseClient'
-```
+We can use the [Supabase Auth UI](/docs/guides/auth/auth-helpers/auth-ui), a pre-built Svelte component, for authenticating users via OAuth, email, and magic links.
+
+Install the Supabase Auth UI for Svelte
 
-### Set up a Login component
+```bash
+npm install @supabase/auth-ui-svelte @supabase/auth-ui-shared
+```
 
-Let's set up a Svelte component to manage logins and sign ups. We'll use Magic Links, so users can sign in with their email without using passwords.
-Create a new `Auth.svelte` component in the `src/routes` directory to handle this functionality.
+Add the `Auth` component to your home page
 
-```html title=src/routes/Auth.svelte
+```svelte title=src/routes/+page.svelte
+<!-- src/routes/+page.svelte -->
 <script lang="ts">
-  import { supabase } from '$lib/supabaseClient'
-
-  let loading = false
-  let email: string
-
-  const handleLogin = async () => {
-    try {
-      loading = true
-      const { error } = await supabase.auth.signInWithOtp({ email })
-      if (error) throw error
-      alert('Check your email for the login link!')
-    } catch (error) {
-      if (error instanceof Error) {
-        alert(error.message)
-      }
-    } finally {
-      loading = false
-    }
-  }
+	import { Auth } from '@supabase/auth-ui-svelte';
+	import { ThemeSupa } from '@supabase/auth-ui-shared';
+	import type { PageData } from './$types';
+
+	export let data: PageData;
 </script>
 
-<form class="row flex-center flex" on:submit|preventDefault="{handleLogin}">
-  <div class="col-6 form-widget">
-    <h1 class="header">Supabase + SvelteKit</h1>
-    <p class="description">Sign in via magic link with your email below</p>
-    <div>
-      <input class="inputField" type="email" placeholder="Your email" bind:value="{email}" />
-    </div>
-    <div>
-      <input type="submit" class="button block" value={loading ? 'Loading' : 'Send magic link'}
-      disabled={loading} />
-    </div>
-  </div>
-</form>
+<svelte:head>
+	<title>User Management</title>
+</svelte:head>
+
+<div class="row flex-center flex">
+	<div class="col-6 form-widget">
+		<Auth
+			supabaseClient={data.supabase}
+			view="magic_link"
+			redirectTo={`${data.url}/logging-in?redirect=/`}
+			showLinks={false}
+			appearance={{ theme: ThemeSupa, style: { input: 'color: #fff' } }}
+		/>
+	</div>
+</div>
 ```
 
-### Account component
+Create a `src/routes/+page.server.ts` file that will return our website url to be used in our `redirectTo` above.
 
-After a user is signed in, they need to be able to edit their profile details and manage their account.
-Create a new `Account.svelte` component in the `src/routes` directory to handle this functionality.
+> This is necessary because the current supabase auth flow uses implicit grant which returns the tokens as part of the url fragment (#), 
```

Comment:
```suggestion
> This is necessary because the current Supabase auth flow uses implicit grant which returns the tokens as part of the url fragment (#), 
```